apis/quarkus-common/src/main/java/io/stargate/sgv2/common/grpc/proto/Rows.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.common.grpc.proto;
```

Comment:
As I'm copying this I realize the naming is not very consistent, for example `Rows` and `SchemaReads` serve a similar purpose (helper methods on top of the generated proto stuff), so they should probably be in the same package.
I'm open to a global refactoring once we have everything we need in this module.